src/web/components/BusinessContacts/BusinessContact.tsx

```diff
@@ -0,0 +1,57 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { BusinessContactForm, BusinessContactResponse } from '../../services/participant';
+import BusinessContactDialog from './BusinessContactDialog';
+
+type BusinessContactProps = {
+  contact: BusinessContactResponse;
+  onBusinessContactUpdated: () => void;
+  removeEmailContact: (id: number) => Promise<void>;
+  updateEmailContact: (id: number, form: BusinessContactForm) => Promise<void>;
+};
+
+function BusinessContact({
+  contact,
+  onBusinessContactUpdated,
+  removeEmailContact,
+  updateEmailContact,
+}: BusinessContactProps) {
+  const onRemoveEmailContact = async () => {
+    await removeEmailContact(contact.id);
+    onBusinessContactUpdated();
+  };
+
+  const onUpdateEmailContact = async (formData: BusinessContactForm) => {
+    await updateEmailContact(contact.id, formData);
+  };
+
+  return (
+    <tr>
+      <td>{contact.name}</td>
+      <td>{contact.emailAlias}</td>
+      <td>{contact.contactType}</td>
+      <td className='action'>
+        <BusinessContactDialog
+          onFormSubmit={onUpdateEmailContact}
```

Comment:
These names are confusing - it sounds like they're just the same thing. I can see that one happens before the update and one after, but this seems unnecessarily complex and easy for someone to misunderstand what's happening.

If you're trying to signal that the data is out of date and the top level should revalidate everything ... surely the top level already knows that because it's making the network call?